### DISCUSSION TITLE
species of "addMotifAnnotations"

### DISCUSSION THREAD

**Discussion by chen-zhan** [GreenleafLab/ArchR#1709] [Questions / Documentation]
Hi, I was confused when I use "addMotifAnnotations". I got same error.
`> proj <- addMotifAnnotations(ArchRProj = proj, motifSet = "cisbp", name = "Motif", species="drosophila melanogaster")
ArchR logging to : ArchRLogs/ArchR-addMotifAnnotations-421756ec692a-Date-2022-11-01_Time-10-14-54.log
If there is an issue, please report to github with logFile!
2022-11-01 10:15:06 : Getting Motif Set, Species : drosophila melanogaster, 0.166 mins elapsed.
Error in addMotifAnnotations(ArchRProj = proj, motifSet = "cisbp", name = "Motif",  :
  Species not recognized homo sapiens, mus musculus supported by CisBP!
`
We used the genome built by ourselves. So I don't know how run it

**Comment by rcorces** [comment]
I have updated this error message to be more clear. Only human and mouse motif sets are available through CisBP because these motif sets are pulled from chromVAR. For drosophila, you will have to use a different `motifSet`

update to error message on `dev`:
https://github.com/GreenleafLab/ArchR/commit/bf7613c31896fbb1d7c3d61258a432f2e4e319fd
